### PR TITLE
Make storybook optional

### PR DESCRIPTION
**Pull request by sungwoncho**

### CODE REVIEW

**Comment by macrozone** [comment]
awesome, looks good!

File: test/commands/create_test.js [hunk 1]
```diff
@@ -17,30 +48,23 @@ describe("create command", function() {
...
-    expect(checkFileOrDirExists('./blog/lib/collections/index.js')).to.equal(true);
-    expect(checkFileOrDirExists('./blog/server/publications/index.js')).to.equal(true);
-    expect(checkFileOrDirExists('./blog/server/methods/index.js')).to.equal(true);
-    expect(checkFileOrDirExists('./blog/server/configs/')).to.equal(true);
-    expect(checkFileOrDirExists('./blog/server/main.js')).to.equal(true);
-    expect(checkFileOrDirExists('./blog/.eslintrc')).to.equal(true);
-    expect(checkFileOrDirExists('./blog/.babelrc')).to.equal(true);
-    expect(checkFileOrDirExists('./blog/.scripts/mocha_boot.js')).to.equal(true);
-    expect(checkFileOrDirExists('./blog/mantra_cli.yaml')).to.equal(true);
+    checkCommonFiles();
```
Comment (macrozone) [inline test/commands/create_test.js:51]: nice refactoring

File: lib/utils.js [hunk 2]
```diff
@@ -7,6 +7,30 @@ import {mkdirsSync, outputFileSync} from 'fs-extra';
...
+};
+
+/**
+ * generateConfig generates a custom config object based on the default config
+ * and overriding values.
+ *
+ * @param customConfig {Object} - key values pairs to override the default config
+ * @return {Object} - custom config
+ */
+function generateConfig(overrides) {
```
Comment (macrozone) [inline lib/utils.js:22]: that is a really good idea!

File: test/commands/create_test.js (hunk 1 above)
Comment (ghost) [reply test/commands/create_test.js:51]: Yeah it is. 3 vs 1 :smile:

**Pull request merged**